Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
+description: Use Demisto server REST APIs
+configuration:
+- display: Demisto Server URL
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Demisto Server API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: User system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |
+    var serverURL = params.url;
+    if (serverURL.slice(-1) === '/') {
+        serverURL = serverURL.slice(0,-1);
+    }
+
+    var sendRequest = function(method, url, body) {
+        var requestUrl = serverURL;
+        if (url.slice(-1) != '/') {
```

Comment:
`url.charAt(0) !== '/'`

Reply:
not sure what's the diff

Reply:
![image](https://user-images.githubusercontent.com/5429033/33019732-43a53ab2-ce04-11e7-838a-56539cd37182.png)

Reply:
![image](https://user-images.githubusercontent.com/5429033/33019745-54491ae6-ce04-11e7-91e9-0887c10238e9.png)

Reply:
but i'm checking the last char.